templates/vars/vnet/cftptl-intsvc.json

```diff
@@ -129,6 +129,22 @@
           "service": "Microsoft.AzureCosmosDB"
         }
       ]
-    }
+    },
+    "routetableRoutes": {
+      "value": [
+          {
+          "name": "Internet",
+          "nextHopType": "VirtualAppliance",
+          "nextHopIpAddress": "10.11.8.36",
+          "addressPrefix": "0.0.0.0/0"
+        },
+        {
+          "name": "sdsdev",
```

Comment:
why is this needed? won't the hub route catch this?

Reply:
Was requested by Luigi

Reply:
@luigibk why is the specific route needed?

Reply:
Grafana

Reply:
won't it work anyway once it uses a default route to the hub?

Reply:
Until it's overridden by something propagated from somewhere as we've seen quite often recently ... yes

Reply:
can we remove and only add if required then? it would be better to rely on default route to hub as much as possible imo

Reply:
That depends really, but remove it for now and it can always be added again later

Reply:
Cheers both - now removed.